maintainers/maintainer-list.nix

```diff
@@ -13589,4 +13589,15 @@
     github = "PhilippWoelfel";
     githubId = 19400064;
   };
+  qbit = {
```

Comment:
Welcome aboard!

The info about you should be put in a separated commit. Please, split accordingly.

Reply:
Thanks! fixed!